src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  function onChangeTabHandler(event, index) {
+    event.preventDefault();
+    onTabSelected(tabs[index]);
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map((tab, index) => (
+              <li
+                className={selectedTab.id === tab.id ? 'is-active' : null}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={e => onChangeTabHandler(e, index)}
```

Comment:
```suggestion
                  onClick={() => onChangeTabHandler(tab)}
```

Reply:
i taking index of tabs array element and setting it as a new tab[index]

i dont understood why i need to set tab.id as initial state? it is not comfortable at all.

Reply:
will be grateful if you can explain me this logic. 
i saw at working version that initial state in it set tot tab.id

Reply:
I'm not sure i understood you.
in this case we set needed tab directly without any manipulation with indexes or smth like that